cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreTableManager.java

```diff
@@ -375,20 +375,25 @@ private String generateFileSetCreateStatement(Id.DatasetInstance datasetID, Data
 
     String format = FileSetProperties.getExploreFormat(properties);
     if (format != null) {
-      // for text and csv, we know what to do
-      Preconditions.checkArgument("text".equals(format) || "csv".equals(format),
-                                  "Only text and csv are supported as native formats");
-      String schema = FileSetProperties.getExploreSchema(properties);
-      Preconditions.checkNotNull(schema, "for native formats, explore schema must be given in dataset properties");
-      String delimiter = null;
-      if ("text".equals(format)) {
-        delimiter = FileSetProperties.getExploreFormatProperties(properties).get("delimiter");
-      } else if ("csv".equals(format)) {
-        delimiter = ",";
+      if ("PARQUET".equals(format)) {
+        return createStatementBuilder.setSchema(FileSetProperties.getExploreSchema(properties))
+          .buildWithFileFormat("PARQUET");
+      } else {
+        // for text and csv, we know what to do
+        Preconditions.checkArgument("text".equals(format) || "csv".equals(format) || "PARQUET".equals(format),
```

Comment:
I just used the instruction in hive 0.13 to create a parquet dataset, it is upper case there, and the statement is like:
CREATE TABLE parquet_test (
 id int,
 str string,
 mp MAP<STRING,STRING>,
 lst ARRAY<STRING>,
 strct STRUCT<A:STRING,B:STRING>) 
PARTITIONED BY (part string)
STORED AS PARQUET;